_posts/2020-10-10-temperature-water-content-and-polishing-effects-on-koji.md

```diff
@@ -0,0 +1,134 @@
+- rice
+- featured
+- water-content
+---
+
+I always wondered if researchers had played around with koji growing parameters - turns out they have. This post will be the first of a few going over these studies - starting with temperature, water content, and polishing rate.
+
+### Overview
+
+Koji-making is all about optimizing growth conditions for the specific application you're after. Although koji-makers had centuries of knowledge about optimal growth conditions, it wasn't examined  scientifically until the 20th century.
+
+Everything in this post comes from the Brewing Society of Japan, and its academic journal (which has been running from 1906 to today)  - more on that at the end of the post. The experiments below were run mostly in the the 1970s and the 1980s by researchers looking to explain and quantify different aspects of koji-making.
+
+### Temperature
+
+One of the most obvious parameters that researchers tested was the **cultivation temperature**. The growth of A. oryzae on white rice was tested at temperatures between 30°C and 40°C - and this is what they found:
+
+![Table 1](/assets/images/temp_effect/Table 1.png)
+*Effect of culture temperature on enzyme activity ([1] - 10mL O<sub>2</sub> consumed/ g koji and [2] at maximum cell mass)*
+
+Of course, the researchers know that growing koji for a fixed time at different temperatures would not be a fair comparison. At different temperatures, the koji's growth rate (μ, measured in h<sup>-1</sup>) is also different. Thus, enzyme activity was compared at two points different points:
+
+1. When the koji reaches 10mL oxygen/g dry weight of koji, and;
+2. When the koji reaches a maximum cell mass
+
+*Conclusion:* in general, higher temperatures favour amylases, and lower temperatures favour proteases and peptidases.
+
+In practice, no koji makers would set a single temperature for their koji for the entire duration of its growth. The above temperatures are mostly important during the last 12 hours of growth when the mycelium is producing the most enzymes.
+
+As an aside, this was the experimental set-up:
+
+![Figure 1](/assets/images/temp_effect/Figure 1.png){:.responsive .center}
+*The experimental setup - see Ref. 2*
+
+*Callout:* This setup is a sort of respirometer - I'm not sure of the exact name. Oxygen is produced by electrolysis, which builds up pressure in the system, pushing one side of the mercury manometer down. With enough pressure, the mercury level switch opens, stopping the generation of oxygen. The koji continues consuming oxygen, and one can measure total oxygen consumption by integrating the total current consumed over time. Finally, waste carbon dioxide is absorbed by the NaOH solution.
+
+### Water content
+
+Another parameter they tested is the **steamed rice water absorption rate**, which is defined as follows:
+
+$$
+\text{Steamed rice water absorption rate (%)} = \frac{\text{Steamed rice weight} - \text{White rice weight}}{\text{White rice weight} }   \times 100\%
```

Comment:
Thoughts on "Dry rice weight" instead of "White rice weight"?

Reply:
Maybe: uncooked rice weight?
Agreed white rice weight is confusing. The key is that it is the weight of "raw" rice i.e. straight out of the bag
I want to avoid "dry rice weight" which may be confused with "dry weight" (0% moisture)